lib/baldr/parser.rb

```diff
@@ -18,16 +18,34 @@ def successful?
     @error.nil?
   end
 
+  def bare_transaction?
+    @root_type == :transaction
+  end
+
+  def each &block
```

Comment:
@npersad84 we weren't passing the envelope, which contains the EDI version (4060), to the broker. The broker needs to recognize the proper grammar